spock-specs/src/test/groovy/org/spockframework/mock/AutoAttachExtensionSpec.groovy

```diff
@@ -0,0 +1,39 @@
+package org.spockframework.mock
+
+import org.spockframework.EmbeddedSpecification
+import org.spockframework.runtime.SpockException
+
+class AutoAttachExtensionSpec extends EmbeddedSpecification {
+
+  def "null field"() {
+    when:
+    runner.runWithImports("""import spock.mock.AutoAttach
```

Comment:
I'd add the import to the runner/compiler in `setup`

Reply:
Done.